### DISCUSSION TITLE
Can i customize globaleaks directly by editing its source code directly in production?

### DISCUSSION THREAD

**Discussion by JimpoTEDY** [globaleaks/globaleaks-whistleblowing-software#4554] [Q&A] [answered]
Hello, unfortunately I cannot find the footer and header.html in File Zilla after the installation. How can I access it? Unfortunately, I don't have that much knowledge about this topic. Thanks in advance.

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your question.

Actually Globaleaks is not meant to be manually customized by editing
files. That would actually invalidate the security of the system.

If you would like please elaborate what you would like to do and we may try
to support you implementing it via the functionalities offered by the web
interface.

**Comment by zangels** [comment]
Hello!
I'm not a coder also but I would like for the interface to be more customizable.

If the original css file was available (maybe it is but I don't know where to get it ) it would be easier to change it and then upload it through the web interface.

A few days ago I request to join to transifex project, to help in the language issues (PT-PT), but there's no anwser.

The forum has not been avalable also, and maybe some of my questions about the customization of the interface or about 'Questionnaires' 'Question templates', have already been anwsered there.

Thank you, and all the team, for all the work.

**Comment by evilaliv3** [comment]
Thank you @zangels 

Here the answers to your comments:
- Actually original files of the application are not meant to be customized but you can upload your custom css via the web interface.
- Thank you, i will ping the language coordinator that support us on transifex and ask them if they could reach out to you.
- We are doing some maintenance to the forum that will be restored shortly. I will update you as soon that it's back up and if you have any question please reach out.

**Comment by evilaliv3** [comment]
@zangels: Which is you username on transifex?

**Comment by zangels** [comment]
@evilaliv3 it's "jose.anjos"
Thkz

**Comment by zangels** [comment]
> * Actually original files of the application are not meant to be customized but you can upload your custom css via the web interface.
@evilaliv3 
but is it possible to have the 'class' name given to the elements?
so I could use them in a/the new css file?

**Comment by evilaliv3** [comment]
Sure thing.

Actually i would invite you to check current classes and ids assigned to HTML elements and let us know if you need anything that you could not customize and make suggestions.

**Comment by zangels** [comment]
> 

Thank you so much!
And where can I do that? or how?

**Comment by evilaliv3** [comment]
By using the javascript console of a common browser

E.g. try to digit CTRL + SHIFT + J on Google Chrome
